rootfs/etc/confd/templates/builder

```diff
@@ -64,17 +64,30 @@ TMP_DIR=$(mktemp -d -p $BUILD_DIR)
 
 cd $REPO_DIR
 # use Procfile if provided, otherwise try default process types from ./release
+
+git archive --format=tar.gz ${GIT_SHA} > ${APP_NAME}.tar.gz
```

Comment:
> I did a ls of files inside repo direcotry and it didn't show up any files related to example-go

It'll all be hiding under the build dir, which will be in `/home/git/example-go/build/<some-random-build-id>`. That's where it's extracted for the build, and that's how we can detect if a Dockerfile is present.